test/unit/index.test.js

```diff
@@ -116,6 +117,23 @@ describe('idx-js', () => {
         });
     });
 
+    it('returns an idxState when an error occur is returned via introspect', async () => {
```

Comment:
Typo - will fix before merging:

```suggestion
    it('returns an idxState when an error is returned via introspect', async () => {
```